tests/new_unit_tests/Commands/SSHKeys/SSHKeysListCommandTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * @file
+ */
+
+namespace Pantheon\Terminus\UnitTests\Commands\Auth;
+
+
+use Pantheon\Terminus\Commands\SSHKey\ListCommand;
+use Pantheon\Terminus\Config;
+use Pantheon\Terminus\UnitTests\Commands\SSHKeysCommandTest;
+use Terminus\Collections\SshKeys;
+use Terminus\Models\SshKey;
+
+class SSHKeysListCommandTest extends SSHKeysCommandTest {
+
+  /**
+   * Sets up the fixture, for example, open a network connection.
+   * This method is called before a test is executed.
+   */
+  protected function setUp()
+  {
+    parent::setUp();
+
+    $this->command = new ListCommand(new Config());
```

Comment:
If it pleases you to inherit from `\Pantheon\Terminus\UnitTests\Commands\CommandTestsCase`, you can automatically inherit a `$this->config()`. It would be nice to have all the command tests inheriting the same parent class, but if it is a hinderance to you for some reason, please let me know so it can be addressed.